ecs-cli/modules/cli/compose/project/project_version.go

```diff
@@ -0,0 +1,47 @@
+package project
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	yaml "gopkg.in/yaml.v2"
+)
+
+func (p *ecsProject) checkComposeVersion() (string, error) {
+	var composeVersion string
+	if len(p.ecsContext.ComposeFiles) == 0 {
+		return "", fmt.Errorf("No Compose files found")
+	}
+	for _, file := range p.ecsContext.ComposeFiles {
+		fileVersion, err := getFileVersion(file)
```

Comment:
Yes, it will. As discussed offline:
 - while '2' and '2.0' represent the same version, current behavior w/ libcompose throws an error if "x.0" is detected. See https://github.com/docker/libcompose/issues/490
 - so, if "x.0" is detected, we can either 1) accept it and proceed, knowing it will error on libcompose parse(), or 2) accept it and modify the compose bytes so that '2' is passed in and accepted by libcompose
 - given the above, we are opting to 1) keep the consistency check since it will write out the detected versions, making the 2/2.0 discrepancy known 2) not modify the bytes passed to libcompose, but log out a warning that x.0 versions are not valid. This way the error will surface from libcompose (where the problem originates) but we provide better messaging re: the cause